Hic scaffolding user guide (#87)
AustralianBioCommons/how-to-guides: genome_assembly/hic-scaffolding.md

---
title: Genome scaffolding with Hi-C on Galaxy Australia

> type: genome-scaffolding

Reviewer: ```suggestion
type: genome-assembly
```

> contributors: [VGP Project, Luke Silver, Anna Syme]

Reviewer: Should `VGP` be an affiliation instead?

Author: I think contributor is better, as they are really the authors of the base workflow.

description: This guide describes the steps required to scaffold your genome on the Galaxy Australia platform using HiC data
affiliations: [Bioplatforms Australia, Galaxy Australia, Australian BioCommons, Threatened Species Initiative]
---

> This How-to-Guide will describe the steps required to scaffold your genome on the Galaxy Australia platform, using HiC data, developed in consultations between the Bioplatforms Australia [Threatened Species Initiative](https://threatenedspeciesinitiative.com/), [Galaxy Australia](https://usegalaxy.org.au/), and the [Australian BioCommons](https://www.biocommons.org.au/).

Reviewer: ```suggestion
This How-to-Guide will describe the steps required to scaffold your genome on the Galaxy Australia platform using a HiC scaffolding workflow developed in consultations between the Bioplatforms Australia [Threatened Species Initiative](https://threatenedspeciesinitiative.com/), [Galaxy Australia](https://usegalaxy.org.au/), and the [Australian BioCommons](https://www.biocommons.org.au/).
```

> This workflow has been created from a Vertebrate Genomes Project (VGP) scaffolding workflow. 

Reviewer: Link to VGP?

Author: Yes, have added.

* For more information about the VGP project see the [Galaxy-VGP project page](https://galaxyproject.org/projects/vgp).

> * The VGP scaffolding workflow is hosted on [Dockstore](https://dockstore.org/workflows/github.com/iwc-workflows/Scaffolding-HiC-VGP8/main:main?tab=info). Please see that link for the workflow diagram and more information if required. 

Reviewer: Also: https://workflowhub.eu/workflows/625

Author: Changed this one.

{% include callout.html type="note" content="If you need help, the Galaxy community is both approachable and helpful. [Ask them questions!](https://help.galaxyproject.org/)" %}

### Register and login

1. To register for Galaxy Australia, visit the [login page](https://usegalaxy.org.au/login).
2. Click the ```Register here``` link.
3. Complete the registration wizard and click ```Create```.
4. Login to your account!

### Upload data files

Please see the how-to guide for HiFi genome assembly for additional information about uploading data from the Bioplatforms Australia Data Portal.

* In Galaxy Australia, create a new history
* Import data
    * ```assembly.fasta```. This file may be in one of your Galaxy histories.
    * HiC data: concatenated ```HiC_F.fastqsanger.gz```, concatenated ```HiC_R.fastqsanger.gz```
  
### Import the scaffolding workflow

Please see the how-to guide for HiFi genome assembly for additional information about how to import and run workflows. A brief summary is given here:

* In Galaxy, go to the Workflow tab
* Click "Import"
* Click on the tab at the right, "TRS ID"
* Make sure the "TRS server" is set to `workflowhub.eu`
* At "TRS ID", type in `1054`

> * Click on the version you wish to import (e.g. Version 1)

Reviewer: This could be replaced with a TRS link, like the other guides: e.g. https://github.com/AustralianBioCommons/how-to-guides/blob/main/genome_annotation/Transcriptome.md#run-rna-seq-qc-and-trimming-workflow

This would make it much easier for a user to access the workflow.

Author: Yep, changed this.

    * restriction enzymes
    * HiC forward and reverse reads - these need to be a single concatenated file for each set, and in fastqsanger.gz format
    * Estimated genome size as integer
    * Lineage for BUSCO
* Click Run

### Check the outputs

The main outputs are:
* scaffolded_assmbly.fasta
* comparison of pre- post- scaffolding contact maps

For more information about what these outputs mean, please see the HiC Scaffolding section in the [VGP assembly tutorial](https://training.galaxyproject.org/training-material/topics/assembly/tutorials/vgp_genome_assembly/tutorial.html).

 ## Acknowledgements

* We acknowledge and thank the Vertebrate Genomes Project for this workflow. This is not original work, but extracted from the VGP scaffolding workflow with some minor changes to inputs to better support various groups of researchers. More information about VGP workflows: [Galaxy-VGP project page](https://galaxyproject.org/projects/vgp).
* Bioplatforms Australia [Threatened Species Initiative](https://threatenedspeciesinitiative.com/)
* [Galaxy Australia](https://usegalaxy.org.au/)
* The [Australian BioCommons](https://www.biocommons.org.au/).

Workflow citation: 

> > Syme, A., Silver, L., based on VGP Project (2024). **TSI-Scaffolding-with-HiC (based on VGP-HiC-scaffolding)**. WorkflowHub. <a href="https://doi.org/10.48546/WORKFLOWHUB.WORKFLOW.1054.1" ga-product="workflow_doi" ga-id="1054_conversion_qc">https://doi.org/10.48546/WORKFLOWHUB.WORKFLOW.1054.1</a>

Reviewer: ```suggestion
> Syme, A., Silver, L., based on VGP Project (2024). **TSI-Scaffolding-with-HiC (based on VGP-HiC-scaffolding)**. WorkflowHub. [https://doi.org/10.48546/WORKFLOWHUB.WORKFLOW.1054.1](https://doi.org/10.48546/WORKFLOWHUB.WORKFLOW.1054.1)
```
